### PR TITLE
[web] move some vector functions into vector_math.dart

### PR DESCRIPTION
**Pull request by yjbanov**
Move `toMatrix32` and the (not so) `FastMatrix32` vector math utilities into the `vector_math.dart` library, where they belong.

### CODE REVIEW

File: lib/web_ui/lib/src/engine/html/scene_builder.dart [hunk 1]
```diff
@@ -6,7 +6,7 @@ import 'dart:typed_data';
 
 import 'package:ui/ui.dart' as ui;
 
-import '../../engine.dart' show kProfileApplyFrame, kProfilePrerollFrame, toMatrix32;
```
Comment (ditman) [inline lib/web_ui/lib/src/engine/html/scene_builder.dart:9]: Didn't we have a lint for unused imports?

**Review by ditman — APPROVED** [review]
I would exempt this of tests, it is just moving already existing code around. These are utility functions that are tested indirectly through other code that handles vector transformations elsewhere.

File: lib/web_ui/lib/src/engine/html/scene_builder.dart (hunk 1 above)
Comment (yjbanov) [reply lib/web_ui/lib/src/engine/html/scene_builder.dart:9]: We do, but I think the lint doesn't trigger when two libraries vend the same member.

**Comment by ditman** [comment]
Tests were added.

**Pull request merged**